utils/src/main/kotlin/DeclaredLicenseProcessor.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.utils
```

Comment:
Why is this in the "utils" module (not package) instead of "spdx-utils" (as it is very SPDX-specific) or in the "analyzer" module (as that's the only place where it is used)?

Reply:
I had it in analyzer first, but then thought I would use it directly from model classes and moved it to utils. However, it turned out that I cannot use it from model classes directly because of error handling, so it could go back to the analyzer now. However, the copyright statement processor is also in utils, so it would also make sense to keep both processors in the same module.
I could not add it to spdx-utils, because `log` is not available there.

Reply:
Ok, already having the copyright statement processor in utils convinced me.